uqcsbot/scripts/events.py

```diff
@@ -138,9 +149,14 @@ def __str__(self):
         else:
             end_str = f"{d2.hour}:{d2.minute:02}"
 
+        if self.recurring:
+            rec_str = "[RECURRING] "
+        else:
+            rec_str = ""
+
         # Encode user-provided text to prevent certain characters
         # being interpreted as slack commands.
-        summary_str = Event.encode_text(self.summary)
+        summary_str = Event.encode_text(rec_str + self.summary)
```

Comment:
Could be replaced with a constant `RECURRING = "[RECURRING]"` and `Event.encode_text(f"{RECURRING} " if self.recurring else "" + self.summary)`

Reply:
Yeah that is much neater